### ISSUE TITLE
Janky Scrolling

### ISSUE THREAD

**Issue by ScriptRaccoon** [LeaVerou/lea.verou.me#9]
### Page URL:

lea1.verou.me

### What issues do you see?

Other

### Description of issues

On my phone (Google Pixel 5, Android) the website scrolling is choppy. I tested with Adblock browser (1st recording) and Chrome (very similar, but slightly better). It does not happen with other, similar portfolio websites and is unrelated to internet speed.
In the previous website, scrolling was much more choppy (2nd recording).

**Comment by LeaVerou** [comment]
I think this is probably related to the background image animation on the header and footer. That would also explain why it’s less choppy now: Instead of making it play constantly when the element is in view, it now is tied to the scroll position.

I wonder if there’s a way to detect the devices where this would affect scrolling speed (e.g. in my MacBook Pro it doesn’t seem to) and disable it then.
Also, once [`@scroll-timeline`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-timeline) gets wider support it may also allow improving this.

**Comment by ScriptRaccoon** [comment]
Indeed, when I remove the footer and the header, the scrolling works much better. There is still another issue then (the "learn about me" button is flashing), but it is definitely worth a try to look there.
By the way, I tried to clone your repo and run the dev server. The console looks fine, Netlify works, but the browser gives me a 404 page. So I am afraid that, for now, I cannot contribute.

**Comment by opatry** [comment]
I can confirm this behavior on my phone too.

**Comment by opatry** [comment]
A bit less but yes, still not very smooth.

**Comment by LeaVerou** [comment]
Another question: Is this different in posts that don't have a table of contents, e.g. https://lea.verou.me/blog/2021/06/inherit-ancestor-font-size-for-fun-and-profit/ ?

**Comment by SarasovMatvey** [comment]
Same problems with scrolling

https://github.com/user-attachments/assets/5f434b59-df13-4090-8286-5570299e83c7
